bandwidth-file-spec.txt

```diff
@@ -358,7 +491,7 @@
   If a parser does not recognize any extra material in a RelayLine,
   the extra material MUST be ignored.
 
-  Each RelayLine MUST include the following KeyValue pairs:
+  Each RelayLine includes the following KeyValue pairs:
 
     "node_id=" hexdigest
```

Comment:
Why did you remove all the "SP" in the lines?. 
I'm not sure this can be deduced from the definitions in section 2.1

Reply:
The definition of RelayLine is:
`RelayLine ::= KeyValue (SP KeyValue)* NL`
Which means that there is a space after every KeyValue, except the last KeyValue.